room.cpp

```diff
@@ -612,6 +616,39 @@ void Room::postMessage(const RoomMessageEvent& event)
     connection()->callApi<SendEventJob>(id(), event);
 }
 
+void Room::inviteCall(const QString& callId, const int& lifetime,
+                const QString& sdp)
+{
+      CallInviteEvent rme(callId, lifetime, sdp);
+      connection()->callApi<SendEventJob>(id(), rme);
+}
```

Comment:
The following two points apply to all the `Room` methods you're introducing:
- You're replicating `Room::postMessage()` here. Consider just passing a created event to `postMessage instead` (see line 611 above).
- As also mentioned below, `Room::inviteCall()` must only actually put a call event if the room has exactly two members. Specific way to do validation is up to you; I prefer clients to apply such validations on their side so I'd put a `Q_ASSERT` here (potentially blowing up a misbehaving client). At the very least, there should be a `qCWarning(MAIN)` instead of sending a message event when `memberCount() != 2`.

Reply:
The reason why i created separated function is so they can be called directly from QML, but i can move it to the client side (uMatriks) if this is not the correct place.

Reply:
I meant a bit different thing: the functions are in place; but their bodies can be made one-liners calling postMessage instead of the current two-liners.

Reply:
Scratch that, I  understood the issue. Keep it like now.